octoprint_mrbeam/rest_handler/update_handler.py

```diff
@@ -12,5 +12,12 @@ class UpdateRestHandlerMixin:
 
     @octoprint.plugin.BlueprintPlugin.route("/info/update", methods=["POST"])
     def update_update_informations(self):
-        reload_update_info(self)
-        return NO_CONTENT
+        data = None
+        clicked_by_user = False
+        if hasattr(request, "json") and request.json:
+            data = request.json
+            clicked_by_user = data.get("user", False)
```

Comment:
Are you sure you want to default to False ?

Reply:
Yes, if it is not set the default should be false, as we only set it to true if the users clicks on it and otherwise leave this data element empty